src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -229,7 +229,7 @@ ApplicationManagerImpl::ApplicationManagerImpl(
 ApplicationManagerImpl::~ApplicationManagerImpl() {
   SDL_LOG_AUTO_TRACE();
 
-  is_stopping_ = true;
+  is_stopping_.exchange(true);
```

Comment:
Why using `exchange` instead of `store`?
I think `exchange` could be more suitable in case we wanted to save previous value of `is_stopping_` variable.

Reply:
@YarikMamykin actually, just moved a fix without any changes. However I agree that `store` is more applicable in that case. Replaced in https://github.com/smartdevicelink/sdl_core/pull/3491/commits/8209b59b0d52b22d82ab243a7c0d631de55bd662